web/client/epics/identify.js

```diff
@@ -30,6 +34,17 @@ module.exports = {
         .switchMap(() => {
             return Rx.Observable.of(closeFeatureGrid());
         }),
+    /**
+     * Check if something is editing in feature info.
+     * If so, as to the proper tool to close (annotations)
+     * Otherwise it closes by itself.
+     */
+    closeFeatureInfoOnEdit: (action$, {getState = () => {}} = {}) =>
```

Comment:
I would change the name to a more generic one: this is about closing feature info, the cause is not relevant

Reply:
I agree, I'm always afraid to override existing epic names (they are merged in a unique key - value object before to be applied from the plugins. Maybe we should somehow find out a better way to manage and avoid duplication.